packages/flutter/lib/src/cupertino/bottom_tab_bar.dart

```diff
@@ -156,25 +156,30 @@ class CupertinoTabBar extends StatelessWidget implements PreferredSizeWidget {
     final List<Widget> result = <Widget>[];
 
     for (int index = 0; index < items.length; index += 1) {
+      final bool active = index == currentIndex;
       result.add(
         _wrapActiveItem(
           new Expanded(
-            child: new GestureDetector(
-              behavior: HitTestBehavior.opaque,
-              onTap: onTap == null ? null : () { onTap(index); },
-              child: new Padding(
-                padding: const EdgeInsets.only(bottom: 4.0),
-                child: new Column(
-                  mainAxisAlignment: MainAxisAlignment.end,
-                  children: <Widget> [
-                    new Expanded(child: new Center(child: items[index].icon)),
-                    items[index].title,
-                  ],
+            child: new Semantics(
+              selected: active,
+              hint: 'tab, ${index + 1} of ${items.length}',
```

Comment:
I think this needs to be localized somehow, I don't have a good solution here.

Reply:
We'd have to do #13452 first